inference/serve.py

```diff
@@ -60,8 +71,9 @@ def main(argv=None):
     import argparse
     parser = argparse.ArgumentParser(description="Model Serve Script", add_help=False)
     parser.add_argument("--config_file", type=str, help="inference configuration file in YAML. If specified, all other arguments are ignored")
-    parser.add_argument("--model", default=None, type=str, help="model name or path")
-    parser.add_argument("--tokenizer", default=None, type=str, help="tokenizer name or path")
+    parser.add_argument("--model_id_or_path", default=None, type=str, help="model name or path")
+    parser.add_argument("--tokenizer_id_or_path", default=None, type=str, help="tokenizer name or path")
+    parser.add_argument("--model_to_serve", nargs='*', default=["gpt2"], type=str, help="Only used when config_file and model_id_or_path are both None, it needs to be a subset of the values of key 'name' in inference/models/*.yaml")
```

Comment:
I found gpt2 may not be supported by deepspeed/ipex/vllm. we need to verify that. if that is the case, we need to change default model to gpt-j-6b

Reply:
Now the default value of deepspeed in all configuration files is False. We can keep using gpt2.yaml here for a smooth testing experience. Then users can pass model parameters according to their needs. I think we need to update https://github.com/intel/llm-on-ray/issues/51 in serving docs.